seq2science/rules/peak_calling.smk

```diff
@@ -283,11 +283,15 @@ if "condition" in samples:
             """
             Combine replicates based on the irreproducible discovery rate (IDR). Can only handle two replicates.
             For more than two replicates use fisher's method.
+
+            When combining narrowpeak files we make the q-score zero, and the peak summit in the middle. This is
```

Comment:
typo and a bit vague. Did you mean "...take the q-score, and the center of the peak summit."

Reply:
Nope. IDR makes the qscore and summit both -1, that destroys some downstream tools. So I decided to put the summit in the middle, and the q-score as zero. I'll update the text to be more clear